android_vs_ios.md

```diff
@@ -0,0 +1,175 @@
+This document tries to provide a high level overview of some key differences (and similarities) of developing for Android versus developing for iOS. It is aimed to be a reference for people who are starting to or are considering switching from one mobile platform to another.
+
+# Language
+
+**TL;DR: Kotlin is more convenient, Swift is more powerful**
```

Comment:
Not sure if I agree or disagree with this, but in any case it's not clear to me that it's an accurate summary of what you've written here

Reply:
kotlin adds a lot of features that helps reduce boilerplate such as delegation, object type, data class, scoping function etc, but there's not many features that allows you to do more stuff compared to Java or Swift. Swift allows for a more control over memory management, has value/struct type, runtime generics, and the ability to use extension to add protocol conformance.

Reply:
I see that argument, I'm just not sure you actually make those points very explicitly in the text. But not really a big deal.